src/components/ItaliaTheme/View/Commons/ContactLink.jsx

```diff
@@ -32,70 +31,92 @@ const messages = defineMessages({
 
 const ContactLink = ({ tel, fax, email, label = true, strong = false }) => {
   const intl = useIntl();
-  let ret_label = null;
-  let ret = null;
 
-  function ReplacePhoneNumbers(str, type) {
-    // eslint-disable-next-line no-useless-escape
-    let newhtml = str.replace(/\+?[0-9]( ?[0-9\/-]+)+.?[0-9]*/gm, function (v) {
-      let r =
-        "<a href='" +
-        type +
-        ':' +
-        v.trim().replace(/-|\/|\s/gm, '') +
-        "' title='" +
-        (type === 'tel'
-          ? intl.formatMessage(messages.call)
-          : intl.formatMessage(messages.call_fax)) +
-        "' >" +
-        v +
-        '</a>';
-      return r;
-    });
-    return newhtml;
-  }
+  const formatTel = (str, type) =>
+    str
+      .split(/((?:\+?\d{1,3}[-.\s]?)?(?:\(?\d{3}\)?[-.\s]?)?\d{3}[-.\s]?\d+)/gm)
+      .map((v, i) =>
+        i % 2 === 0 ? (
+          <span key={i}>{` ${v} `}</span>
```

Comment:
non ho capito questa cosa del %2.
cosa succede se in un campo scrivono due numeridi telefono? 
tipo 
"0532123456 0532112233"

Reply:
la split fa appunto split della stringa, tornando alternativamente i match e le stringhe comprese tra i match, nel tuo caso tornerebbe

```
['', '0532123456', ' ', '0532112233', '']
```

non sarebbe male qui metterci degli unit, ...